Esri/arcgis-maps-sdk-swift-samples
# [New] Augment reality to show hidden infrastructure

## Description

This PR implements `Augment reality to show hidden infrastructure` in `Augmented Reality` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-AugmentRealityToShowHiddenInfrastructure/Shared/Samples/Augment%20reality%20to%20show%20hidden%20infrastructure)

## Linked Issue(s)

- `swift/issues/4970`

## How To Test

- Tap on the map to add points for a pipe.
- Tap the check mark button and enter an elevation offset to add the pipe the map.
- Tap the camera button to see the pipe(s) in AR.

## Screenshots

![augment-reality-to-show-hidden-infrastructure-1](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/e46f5cab-91e1-4e43-8254-cb610c1c9d7d)    ![augment-reality-to-show-hidden-infrastructure-2](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/7dfe1a41-4fbb-4ea5-a364-b6d70161493c)
